src/components/MovieDetails.js

```diff
@@ -6,24 +6,45 @@ class MovieDetails extends Component {
     super(props)
     this.state = {
       id: props.location.state.movie.id,
-      movie: {}
+      movie: {},
+      isLoading: true,
+      error: null
     }
   }
 
   componentDidMount = () => {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.state.id}`)
       .then(response => response.json())
       .then(data =>
-        this.setState({ movie: data.movie }))
+        this.setState({ movie: data.movie, isLoading: false }))
+      .catch(error =>
+        this.setState(error))
   }
 
   render() {
+    const {movie, isLoading, error} = this.state;
+
+    if(error) {
+      return <p>{error.message}</p>
```

Comment:
Be sure to show users an error message they will know what to do with - sometimes error messages are more for the FE developer than for the end user.